distribution/openrct2.d.ts

```diff
@@ -3869,7 +3869,7 @@ declare global {
         "map_north_pressed" | "map_south" | "map_south_pressed" | "map_west" | "map_west_pressed" |
         "mountain_tool_even" | "mountain_tool_odd" | "multiplayer" | "multiplayer_desync" | "multiplayer_sync" |
         "multiplayer_toolbar" | "multiplayer_toolbar_pressed" | "mute" | "mute_pressed" | "news_messages" |
-        "normal_selection_6x6" | "paste" | "path_railings" | "path_surfaces" | "paths" | "placeholder" |
+        "normal_selection_6x6" | "palette_invisible" | "palette_invisible_pressed" | "paste" | "path_railings" | "path_surfaces" | "paths" | "placeholder" |
```

Comment:
Its not really the palette here is just an icon to represent the palette. Are we confident that people will understand this?

Reply:
perhaps I could change it to `palette_select_invisible`, `palette_select_invisible_pressed`?

Reply:
applied the previously mentioned change.